### PR TITLE
CAP-36 Add message hash to RadixConnectPassword

### PR DESCRIPTION
**Pull request by danvleju-rdx**

### CODE REVIEW

**Review by CyonAlexRDX — APPROVED** [review]
LGTM, bump cargo toml version after having merged main :) and then we can merge this

**Comment by CyonAlexRDX** [comment]
Missed test in Swift

**Review by sergiupuhalschi-rdx — APPROVED** [review]

**Pull request merged**
